feature-libs/checkout/scheduled-replenishment/components/checkout-schedule-replenishment-order/checkout-schedule-replenishment-order.component.html

```diff
@@ -71,6 +96,21 @@
         </ng-container>
         <ng-template #isDaily>
           <select
+            *cxFeature="'!a11yScheduleReplenishment'"
+            class="form-control"
+            (change)="changeNumberOfDays($event.target.value)"
+          >
+            <option
+              *ngFor="let nDays of numberOfDays"
+              [value]="nDays"
+              [selected]="nDays === scheduleReplenishmentFormData.numberOfDays"
+            >
+              {{ nDays }}
+            </option>
+          </select>
+          <select
+            *cxFeature="'a11yScheduleReplenishment'"
+            id="order-replenishment-period-type"
```

Comment:
We can introduce `id` attributes without a feature toggle because its not a breaking change.

Reply:
I've just updated it.